.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Push
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - '*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
```

Comment:
I still think this step can be entirely removed, but correct me if I'm wrong :)

Reply:
That wasn't required. removed.